src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
-
+spring.datasource.url=jdbc:postgresql://localhost:5432/company
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.username=postgres
+spring.datasource.password=postgreSQL
```

Comment:
it is not obvious but file should be ended with empty line. See why here https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log